rpki-validator/src/main/java/net/ripe/rpki/validator3/api/trustanchors/TrustAnchorService.java

```diff
@@ -156,16 +159,16 @@ public void managePreconfiguredAndExistingTrustAnchors() {
             });
         }
 
-        storage.readTx0(this::scheduleTasValidation);
-    }
-
-    private void scheduleTasValidation(Tx.Read tx) {
-        log.info("Schedule TA validation that were in the database already");
-        trustAnchors.findAll(tx).forEach(ta -> {
-            if (!validationScheduler.scheduledTrustAnchor(ta)) {
-                log.info("Adding " + ta.getName() + " to the validation scheduler");
-                validationScheduler.addTrustAnchor(ta);
-            }
+        storage.readTx0(tx -> {
+            log.info("Schedule TA validation that were in the database already");
```

Comment:
Schedule validation for TAs that were in the database already?

Reply:
Yes, we either  
get TAs from TAL files and schedule stuff for them 
or 
they are already in the Xodus database and and we just schedule stuff for them

Reply:
It was a spelling nit, should have tagged it with that :)